lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -37,55 +37,73 @@
+                    - Only applys for windows web app.
+                    - mutually exclusive with java_version
+
+            java_version:
                 description:
-                    - Number of workers.
-    net_framework_version:
-        description:
-            - The version used to run your web app if using .NET Framework, e.g., 'v4.0' for .NET 4.6 and 'v3.0' for .NET 3.5
-            - Only applys for windows web app.
+                    - The version used to run your web app if using Java, e.g., '1.7' for Java 7, '1.8' for Java 8.
+                    - Only applys for windows web app.
+                    - mutually exclusive with other suboptions
 
-    java_version:
-        description:
-            - The version used to run your web app if using Java, e.g., '1.7' for Java 7, '1.8' for Java 8.
-            - Only applys for windows web app.
+            php_version:
+                description:
+                    - The version used to run your web app if using PHP, e.g., 5.5, 5.6, 7.0.
+                    - Only applys for windows web app.
+                    - mutually exclusive with java_version
 
-    php_version:
-        description:
-            - The version used to run your web app if using PHP, e.g., 5.5, 5.6, 7.0.
-            - Only applys for windows web app.
+            python_version:
+                description:
+                    - The version used to run your web app if using Python, e.g., 2.7, 3.4.
+                    - Only applys for windows web app.
+                    - mutually exclusive with java_version
 
-    python_version:
-        description:
-            - The version used to run your web app if using Python, e.g., 2.7, 3.4.
-            - Only applys for windows web app.
+            node_version:
+                description:
+                    - The version used to run your web app if using nodejs, e.g., 6.6, 6.9.
+                    - Only applys for windows web app.
+                    - mutually exclusive with java_version
 
-    node_version:
-        description:
-            - The version used to run your web app if using nodejs, e.g., 6.6, 6.9.
-            - Only applys for windows web app.
 
-    linux_fx_version:
+    linux_framework:        
         description:
-            - The runtime stack used for your linux-based webapp, e.g., 'RUBY|2.3', 'NODE|6.6', 'PHP|5.6', 'DOTNETCORE|1.1.0'.
-            - Only applys for linx web app. See https://aka.ms/linux-stacks for more info.
+            - The runtime stack used for your linux-based webapp, e.g., 'RUBY|2.3', 'NODE|6.9', 'PHP|5.6', 'DOTNETCORE|1.1.0', 'JAVA|8.0'.
+            - Only applys for linux web app. See https://aka.ms/linux-stacks for more info.
+            - mutually exclusive with windows_framework
+        suboptions:
+            name:
+                description:
+                    - Name of linux runtime framework. e.g., RUBY, NODE, PHP, DOTNETCORE, JAVA.
+                choices:
```

Comment:
I think if there are choices, then you dont' need to have "e.g."

Reply:
fixed